test/mpi_tests/local_checks.sh

```diff
@@ -1,29 +1,26 @@
 #!/bin/bash
-#SBATCH --time=24:00:00
-#SBATCH --nodes=1
-#SBATCH --job-name=mpi_restart_test
-#SBATCH --reservation=clima
-#SBATCH --mem=32GB
 #SBATCH --ntasks=2
+#SBATCH --job-name=mpi_amip
+#SBATCH --time=24:00:00
+#SBATCH --mem-per-cpu=16G
+#SBATCH --partition=expansion
 
-# TODO: this needs to be updated (+ implement better tests that are caught on Buildkite) #667
-
+export MODULE_PATH=/groups/esm/modules:$MODULE_PATH
 module purge
-module load julia/1.10.1
-export JULIA_MPI_BINARY=system
-export JULIA_NUM_THREADS=${SLURM_CPUS_PER_TASK:=1}
-export CLIMACORE_DISTRIBUTED="MPI"
-export JULIA_HDF5_PATH=""
+module load climacommon/2024_03_18
 
-export RUN_NAME=amip_restart_mpi_test
+export CC_PATH=$(pwd)/
```

Comment:
Here, you are relying on the fact that ClimaCoupler is in PWD. This means that there is only one place in the filesystem where submitting this file would work (e.g., it won't work with `cd /tmp; sbatch ~/ClimaCoupler.jl/test/mpi_tests/local_checks.sh`)

Reply:
Yeah, this is a little hard coded... I'll add a comment as a reminder for the user to check this is what they want. :)

Reply:
You might have to explicitly change the folder when you run with slurm, otherwise pwd might be the folder where you ran sbatch